x-pack/plugins/alerts/server/routes/list_alert_types.test.ts

```diff
@@ -52,7 +53,8 @@ describe('listAlertTypesRoute', () => {
           state: [],
         },
         producer: 'test',
-      },
+        enabledInLicense: true,
+      } as RegistryAlertTypeWithAuth,
```

Comment:
In this file, it's better to do things like `const listTypes: RegistryAlertTypeWithAuth[]` in the above instead. Otherwise it's not fully typed to `RegistryAlertTypeWithAuth` from what I can see. I was able to add extra properties to the object without typecheck failing.